docs/markdown/android/iconbutton.md

```diff
@@ -0,0 +1,114 @@
+    - Displaying multiple IconButtons on a surface that uses the same icon for different actions.
+    - Text is better suited to convey the action and/or the icon isn't quickly recognizable by users.
+    - Destructive, high-emphasis actions, e.g "delete", "remove".
+  </Group>
+</TwoCol>
+
+## Mobile best practices
+
+- Avoid using a floating icon button if it obscures important information on the screen. 
+- IconButton on mobile should primarily utilize the lg (44pt) size as the increased size will better accomodate tapping with a finger.   
+
+For general IconButton best practices, refer to the [IconButton web documentation](/web/iconbutton).
+
+## Accessibility
+
+People use Android’s accessibility features, such as TalkBack and dynamic text sizing to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Material Design and development documentation about accessibility for Android:
+
+[Accessbile design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+[Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
+
+## Variants
+
+### Size
+
+1. **lg** **(48pt)**
+Large is the only size that should be used on Pinner surfaces.
+2. **md** **(32pt)**
+Medium is the size used on more dense UI such as business surfaces or internal tools.
+3. **sm** **(24pt)**
+Small IconButton should be used sparingly and only in places where the UI is very dense.
+
+<ThreeCol>
+<Group center>
+<ImgContainer src="https://i.pinimg.com/originals/30/fa/30/30fa30fd2e78a285efd2b81ef7cba745.png" alt="small IconButton" width={24} height={24} />
+**size = "sm"**
+</Group>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/cd/6b/4f/cd6b4ffcd8254493c5a8da94b1d68e19.png" alt="medium IconButton" width={32} height={32} />
+**size = "md"**
+</Group>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/04/af/d1/04afd13f2e6d2e3a659f133cf1c8b360.png" alt="large IconButton" width={48} height={48} />
+**size = "lg"**
+</Group>
+</ThreeCol>
+
+### Elevation
+
+1. **Flat**
+Flat components sit right on top of the background, with no elevation. Therefore they do not have a drop shadow. 
+2. **Floating**
+Floating components are lifted off of the background, with high-emphasis elevation. They present with a drop shadow which can serve as an affordance for scrolling behavior. 
+
+<TwoCol>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/0f/f4/64/0ff464e1786066e63edc0501c9ad3e1b.png" width={64} height={64} alt="flat IconButton"/>
+**flat**
+</Group>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/74/40/cc/7440cc4536f724c5e6c27fe6eadcb9c8.png" width={64} height={64} alt="floating IconButton"/>
+**floating**
+</Group>
+</TwoCol>
+
+### Styling
+For information on color, icons, roles, and states, refer to the [web IconButton documentation](/web/iconbutton). 
```

Comment:
according to our writing guidelines, no comma after roles...

Reply:
Thanks, fixed!